VAMobile/ios/fastlane/Fastfile

```diff
@@ -118,6 +118,14 @@ platform :ios do
     puts('Begin Demo lane')
   end
 
+  desc "makes a test build to ensure changes to native directories did not break build process. DOES NOT DISTRIBUTE"
+  lane :test_build_only do |options|
+    ensure_xcode_version(version: "15.2", strict: false)
+    update_version(options[:version])
+    sign
+    build
+  end
```

Comment:
Do we plan to use this again in the future?

Reply:
Not sure! But we never know